### DISCUSSION TITLE
How to show Songs menu in the sidebar?

### DISCUSSION THREAD

**Discussion by jsrol** [jeffvli/sonixd#434] [Q&A] [answered]
I am new to Sonixd but really getting to enjoy it. However i cannot for the life of me figure out how to see my songs list and I've looked everywhere.

If i go into Config-> Look and feel -> Sidebar

I see the option "Songs" ticked in blue, but the text and ticker are grayed out as if the function was disabled some other way.

![image](https://user-images.githubusercontent.com/19502683/225970725-1849fe40-4ba1-452f-a545-a105b4b6db6b.png)

I have a massive list of music and i want to be able to quickly check ratings and other track data. How do I get to show my completely unfiltered track list?

**Comment by jeffvli** [comment, marked as answer]
Unfortunately the song list is only available if you're using a Jellyfin server.

Subsonic does not currently have a song list endpoint.

**Comment by jsrol** [comment]
Alright now im even more confused! I just checked Symfonium on my phone and I can see the full song list there. Are they using some trick to bypass limitations? It's connected to the same server. I'm using Navidrome if that makes any difference.

**Reply by jeffvli** [reply]
Symfonium uses the `/search3` endpoint to page through songs, which works but technically isn't the intended usage of the endpoint.

If you need a dedicated song list, I would recommend switching over to [Feishin](https://github.com/jeffvli/feishin), which supports Navidrome's API (though it's still in alpha stages).

**Reply by jsrol** [reply]
I see! 

I just tried Feishin and yeah, it works! I'll hop over discord to bug report along the way.  Seriously good project you got there.

Thanks!
